server/cmwell-cons/src/main/scala/GridSubDiv.scala

```diff
@@ -227,41 +164,38 @@ case class GridSubDiv(user: String,
     val bgAllocations = aloc.bg //DefaultAlocations(1000,1000,512,0)
     val wsAllocations = aloc.ws
     val ctrlAllocations = aloc.ctrl
-
-    val topologyMap = topology.getTopologyMap(ipMappings)
     val homeDir = s"${instDirs.globalLocation}/cm-well"
-    hosts.flatMap { host =>
-      val aliases = topologyMap(host)
+    val casDataDirs = for (i <- 1 to dataDirs.casDataDirs.size)
+      yield {ResourceBuilder.getIndexedName("cas", i)}
```

Comment:
can't we just `(1 to dataDirs.casDataDirs.size).map(ResourceBuilder.getIndexedName("cas", _))` ?

Reply:
done